core/src/main/java/tc/oc/pgm/api/map/includes/MapIncludeProcessor.java

```diff
@@ -0,0 +1,35 @@
+package tc.oc.pgm.api.map.includes;
+
+import java.util.Collection;
+import javax.annotation.Nullable;
+import org.jdom2.Document;
+import tc.oc.pgm.api.Config;
+import tc.oc.pgm.util.xml.InvalidXMLException;
+
+/** A processor to determine which {@link MapInclude}s should be included when loading a map * */
+public interface MapIncludeProcessor {
+
+  /**
+   * Get a {@link MapInclude} which should be included on all maps or null if none.
+   *
+   * @return a {@link MapInclude}
+   */
+  @Nullable
+  MapInclude getGlobalInclude();
```

Comment:
I don't think we need to expose this implementation detail. When `getMapIncludes` on a document, an implementation can include the global. This also makes it easier on the callers of this API.